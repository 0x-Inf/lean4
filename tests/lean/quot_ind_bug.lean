open quotient

variables {A : Type} [s : setoid A] {B : quotient s → Prop} (c : ∀ (a : A), B (quotient.mk a)) (a : A)
check (quotient.ind c ⟦a⟧ : B ⟦a⟧)
check c a
eval (quotient.ind c ⟦a⟧ : B ⟦a⟧)
