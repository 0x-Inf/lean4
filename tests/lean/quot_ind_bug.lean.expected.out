ind c ⟦a⟧ : (λ (_x : quot s), B _x) ⟦a⟧
c a : B ⟦a⟧
c a
